### PR TITLE
Added the ini changes 

### PR DESCRIPTION
**Pull request by ricknjacky**
will be making the new changes soon :) #867

### CODE REVIEW

**Comment by llaske** [comment]
Nice.
But your start branch is still `master` so there are conflicts on your PR (see above)
I suggest to restart from the `dev` branch and reimport your change else it will not be possible to merge.

**Comment by ricknjacky** [comment]
I did the changes you told me to do in the comment on  [867](https://github.com/llaske/sugarizer/issues/867#issuecomment-703224843) added the humane parameter and rechecked my code with the activity too.. waiting for suggestions on further changes to be made. :)

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,14 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+<<<<<<< HEAD
```
Comment (llaske) [inline CHANGELOG.md:7]: This file should not be updated.

File: activities.json [hunk 2]
```diff
@@ -1,4 +1,5 @@
 [
+
```
Comment (llaske) [inline activities.json:2]: This file should not be updated.

File: activities/Moon.activity/css/activity.css [hunk 3]
```diff
@@ -179,4 +179,4 @@ canvas {
 	height: 35px;
 	background-color: #808080;
 	border: 2px solid #808080;
-}
+}
```
Comment (llaske) [inline activities/Moon.activity/css/activity.css:182]: This file should not be updated.

File: activities/Moon.activity/index.html [hunk 4]
```diff
@@ -2,52 +2,53 @@
...
 <head>
-<meta charset="utf-8" />
-<title>Moon</title>
-<meta name="viewport" content="user-scalable=no, initial-scale=1, maximum-scale=1, minimum-scale=1, width=device-width, viewport-fit=cover"/>
-<link rel="stylesheet" media="not screen and (device-width: 1200px) and (device-height: 900px)"
-	href="lib/sugar-web/graphics/css/sugar-96dpi.css">
-<link rel="stylesheet" media="screen and (device-width: 1200px) and (device-height: 900px)"
-	href="lib/sugar-web/graphics/css/sugar-200dpi.css">
-<link rel="stylesheet" href="css/activity.css">
+  <meta charset="utf-8" />
```
Comment (llaske) [inline activities/Moon.activity/index.html:5]: Do not update indentation of the file.
I think the `libnotify.css` file is missing.

**Review by llaske — CHANGES_REQUESTED** [review]
Avoid changing files that do not need to be changed for the PR.
Specifically, change settings of your editor to avoid useless change of indentation in files.
You should not reimplement the `saveImage` behavior, you should just add the `humane.log` call in the current `saveImage` method.

File: activities/Moon.activity/css/activity.css (hunk 3 above)
Comment (ricknjacky) [reply activities/Moon.activity/css/activity.css:182]: I didn't update it it just shows as if it was, initially I thought it should be, you can see it's the same as dev banch

File: activities/Moon.activity/index.html (hunk 4 above)
Comment (ricknjacky) [reply activities/Moon.activity/index.html:5]: ok will check

**Comment by ricknjacky** [comment]
@llaske I have done the changes, still any feedback will be appreciated

ps:-i had to force this commit :( pardon me here.

**Pull request closed without merging**